src/Pages/Order/Order.js

```diff
@@ -0,0 +1,247 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { useLocation } from 'react-router';
+import { useHistory } from 'react-router';
+
+const Order = ({}) => {
+  const [name, setName] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const history = useHistory();
+  const location = useLocation();
+
+  // Option에서 history.push로 받아온 state
+  const {
+    discount_price,
+    room_name,
+    hotel_name,
+    check_in,
+    check_out,
+  } = location.state;
+
+  const handleNameInput = e => {
+    setName(e.target.value);
+    console.log(name);
+  };
+
+  const handleNumInput = e => {
+    setPhoneNumber(e.target.value);
+    console.log(phoneNumber);
+  };
+
+  // 예약하기 클릭 시 fetch POST 통신
+  // alert창으로 예약성공 날려주고 database에 예약 내역 저장 + 예약내역 페이지로 이동
+  const makeOrder = () => {
+    const token = localStorage.getItem('Authorization');
+
+    fetch(`api주소`, {
+      method: 'POST',
+      //  headers 부분이 왜 필요한지..???ㅠㅠ
+      headers: {
+        Authorization: token,
+      },
+      body: JSON.stringify({
+        // 아래 key value는 지원님이랑 실제 통신 시 맞춰봐야함.. +  state로 받아올 수 있는지 확인;;
+        name: name,
+        phone_number: phoneNumber,
+        hotel_name: hotel_name,
+        check_in: check_in,
+        check_out: check_out,
```

Comment:
```suggestion
      body: JSON.stringify({
        name, phone_number, hotel_name, check_in, check_out,
```
- 불필요한 주석은 지워주세요!
- 객체의 key와 value에 들어가는 변수명이 같다면 줄여써줄 수 있습니다 (객체 단축 속성명)